src/sections/maibornwolff.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ojemba</title>
+</head>
+<body>
+    <section class="pb-40">
+        <div class="mt-48 mb-20 ml-36 w-1/3">
+            <h1 class="font-railwayBold text-3xl">MaibornWolff x The Gym x Maliyo</h1>
+            <p class="font-railwayRegular text-lg py-4">
+                Für das Training werden die Erfahrung und das 
+                Wissen über erfolgreiche Softwareprojekte 
+                kombiniert mit dem standortspezifischen ...
+            </p>
```

Comment:
- [ ]  kindly consider breaking this paragraph(adding br tags) as it is in the Figma design to give the expected design